pkg/common/gatewayapi_utils_test.go

```diff
@@ -612,3 +613,111 @@ func TestGatewayWrapperPolicyRefsAnnotation(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestTargetHostnames(t *testing.T) {
+	testRoute := &gatewayapiv1alpha2.HTTPRoute{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "gateway.networking.k8s.io/v1alpha2",
+			Kind:       "HTTPRoute",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "test-route",
+			Namespace: "test-namespace",
+		},
+		Spec: gatewayapiv1alpha2.HTTPRouteSpec{
+			Hostnames: []gatewayapiv1alpha2.Hostname{"*.moog.com", "*.korg.net"},
```

Comment:
I would use always "example.com" domains which are reserved and can be used in literature and testing.

"a.com" and "b.com" I think are also available to be used

Reply:
IANA reserved single letter/digit domains, other than the one that exists, in `.com` that's `q, x & z.com`, they all are good candidates...

Reply:
There goes my tribute heheh. I get it, changed ;)